src/Centreon/Domain/Authentication/UseCase/Logout.php

```diff
@@ -43,12 +44,19 @@ class Logout
      */
     private $authenticationRepository;
 
+    /**
+     * @var SessionRepositoryInterface
+     */
+    private $sessionRepository;
+
```

Comment:
Please could you add the phpDoc for the construct :)

Reply:
done 👍